### ISSUE TITLE
[CSS] Floating values not parsed properly

### ISSUE THREAD

**Issue by contacreast** [sublimehq/Packages#1775]
Ref: https://github.com/sublimehq/Packages/issues/1696

Just got official confirmation from the Chromium team: `1e3px` is valid CSS value for `height` and similar properties that accept pixels or percents. Therefore, it should be implemented in the st text editor.

 Also ping @michaelblyons and @keith-hall who were active in that issue.

**Comment by FichteFoll** [comment]
#1696 already exsists. There is no need for a second one.

**Comment by contacreast** [comment]
No, there is a need for another (this) issue, expecially if we consider that the conversation in the original issue went in totally wrong direction, resulting in non-experienced people arguing about irrelevant stuff. Let @ghost close the original issue and let someone finally fix this.

**Comment by Thom1729** [comment]
There is already an issue requesting the same fix. This issue is redundant. Enjoying the comment thread has nothing to do with it.

**Comment by contacreast** [comment]
Some issues seem to be ignored. Opening a new issue after several months to remind collaborators seems appropriate from my point of view. What about #1610, #1611, #1658, why are they ignored? Leaving a comment in original issue seems not to help and sometimes describing the problem in a new issue seems to be the only option. For the price I paid, I want to see proper highlighting, leaving issue(s) open for several months, while this repo is full of syntax highlighting experts is not appropriate.

**Comment by Thom1729** [comment]
We only need one issue per bug. We already have an issue for this bug. Ergo, this new issue is redundant. Everything you've said is irrelevant.

This entire conversation is off-topic. Hopefully someone will delete it.

If you really want those issues fixed, then fix them. That's why this stuff is on GitHub.

**Comment by contacreast** [comment]
Oh, you are very clever. Nice advice from you, I must admit. So your philosophy is: pay $70, then fix the totally screwed up highlighting engine yourself. Awesome! My point was that some issue are ignored. What is you advice in such case, to wait 5 years hoping that someone will be in a good mood and decide to fix it finally. As said, the three linked java issue are still open, without a single comment for more than 8 months, is it normal? I don't think it is normal. If two issues talking about the same bug are problematic, then why not fix the original one. This issue is more detailed than the first one.

**Comment by contacreast** [comment]
What you linked has nothing to do with this issue, and what I posted to minecraft forge is about some unrelated event from the past. Really don't se what it has to do with this issue.

**Comment by michaelblyons** [comment]
@contacreast If you had added your original post as a comment on #1696, people would have been mildly interested. I don't know that anyone would have opened a PR, but it's frankly a one-line change in your local repo if you _really want_ the `e` to highlight that badly. I might even have helped you do it.

Instead, you are complaining to the people who you want to volunteer for you, saying that the way things work "isn't normal." All kinds of projects have bug reports that are left around unresolved for all sorts of reasons. I'm just a small fish here, but Thom and Fichte are heavy contributors. They're _all_ telling you the same story.
